build.gradle

```diff
@@ -229,4 +229,8 @@ project('oasis-reference'){
         destinationDir(file("$buildDir/deliverable"))
     }
     
+    test {
+        useJUnitPlatform()
+    }
```

Comment:
Read something about fast test stuff, but still going though it with [this course](https://www.baeldung.com/junit-5-migration)